doc/io.xml

```diff
@@ -105,3 +105,80 @@ gap> ReadGenerators(file, 13);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="IteratorFromMultiplicationTable">
```

Comment:
I think this function requires the suffix "File" otherwise the name is misleading.